src/windows/views/timeline_webview.py

```diff
@@ -1183,11 +1183,11 @@ def Layout_Triggered(self, action, clip_ids):
                 clip.data["location_x"] = {"Points": [p_object]}
                 clip.data["location_y"] = {"Points": [p_object]}
 
-            if action == MENU_LAYOUT_CENTER or \
-               action == MENU_LAYOUT_TOP_LEFT or \
-               action == MENU_LAYOUT_TOP_RIGHT or \
-               action == MENU_LAYOUT_BOTTOM_LEFT or \
-               action == MENU_LAYOUT_BOTTOM_RIGHT:
+            if action in [MENU_LAYOUT_CENTER,
+                          MENU_LAYOUT_TOP_LEFT,
+                          MENU_LAYOUT_TOP_RIGHT,
+                          MENU_LAYOUT_BOTTOM_LEFT,
+                          MENU_LAYOUT_BOTTOM_RIGHT]:
```

Comment:
I think `black` would format this as,
```python3
if action in [
        MENU_LAYOUT_CENTER,
        MENU_LAYOUT_TOP_LEFT,
        MENU_LAYOUT_TOP_RIGHT,
        MENU_LAYOUT_BOTTOM_LEFT,
        MENU_LAYOUT_BOTTOM_RIGHT,
        ]:
#\-----/ (note: EIGHT spaces)
```
(I'm not 100% sure where it would put the `]:`, it would either be there or aligned under the `if`.)

...I have a _soft_ preference for that. Not for the usual reasons of enabling cleaner diffs / easier code reviews (the contents of **these** lists just aren't going to change), but more because we frequently end up with conditions so deeply nested and so long that they lead to horrors like,
```python3
                if really_long_variable_name and really_long_variable_name in [MENU_LAYOUT_CENTER,
                                                                               MENU_LAYOUT_TOP_LEFT,
                                                                               MENU_LAYOUT_TOP_RIGHT,
                                                                               MENU_LAYOUT_BOTTOM_LEFT,
                                                                               MENU_LAYOUT_BOTTOM_RIGHT]:
```
That's just the _worst_, and unquestionably (in my mind) should be:
```python3
                if really_long_variable_name and really_long_variable_name in [
                        MENU_LAYOUT_CENTER,
                        MENU_LAYOUT_TOP_LEFT,
                        MENU_LAYOUT_TOP_RIGHT,
                        MENU_LAYOUT_BOTTOM_LEFT,
                        MENU_LAYOUT_BOTTOM_RIGHT,
                ]:
```
or whatever.

But, for **these** that's not a concern, so it really doesn't matter and I wouldn't bother changing it — the aligned-to-the-first-line form is just fine.